router.js

```diff
@@ -0,0 +1,11 @@
+import VueRouter from 'vue-router';
+import Home from './src/components/Home.vue';
+import SearchApp from './src/components/SearchApp.vue';
```

Comment:
Breaking code:
```suggestion
import Articles from './src/components/Articles';
```
Articles is the true parent of the search component -- it brings in the list view.